Update readme (#14)
tier4/pre-commit-hooks-ros: README.md

      - id: prettier-package-xml
      - id: sort-package-xml
```

## Hooks available

### prettier-xacro-xml

Apply prettier to [xacro](http://wiki.ros.org/xacro).

### prettier-launch-xml

Apply prettier to [launch.xml](https://design.ros2.org/articles/roslaunch_xml.html).

### prettier-package-xml

> Apply prettier to [package.xml](https://www.ros.org/reps/rep-0149.html).
> ### sort-package-xml

Reviewer: ```suggestion
Apply prettier to [package.xml](https://www.ros.org/reps/rep-0149.html).

### sort-package-xml
```

      - id: prettier-package-xml
      - id: sort-package-xml
```

## Hooks available

### prettier-xacro-xml

Apply prettier to [xacro](http://wiki.ros.org/xacro).

### prettier-launch-xml

Apply prettier to [launch.xml](https://design.ros2.org/articles/roslaunch_xml.html).

### prettier-package-xml

Apply prettier to [package.xml](https://www.ros.org/reps/rep-0149.html).

### sort-package-xml

Sort the dependent packages in [package.xml](https://www.ros.org/reps/rep-0149.html).

> If you want to exclude a tag from sorting, add `<! -- nolint -->` at the beginning of the line.

Reviewer: ```suggestion
If you want to exclude a tag from sorting, add `<! -- no format -->` at the beginning of the line.
```

```xml

> <!-- nolint --> <depend>rclcpp</depend>

Reviewer: [Nits] It's not `lint`.

```suggestion
<!-- no format --> <depend>rclcpp</depend>
```

      - id: prettier-package-xml
      - id: sort-package-xml
```

## Hooks available

> ### `prettier-xacro-xml`

Author: OK. a6cfc03

> ### `prettier-xacro-xml`

Reviewer: Ah, it was from here.
https://github.com/pre-commit/pre-commit-hooks#hooks-available

> ### `prettier-xacro-xml`

Author: Yes. But it's not necessary.

Apply prettier to xacro.

### `prettier-launch-xml`

Apply prettier to launch.xml

### `prettier-package-xml`

Apply prettier to package.xml
### `sort-package-xml`

Sorts the names contained in each tag in package.xml.

> If you want to exclude a tag from the sort, add `<! -- nolint -->` at the beginning of the line.

Reviewer: I'm not so confident, but I feel this sounds better.

```suggestion
If you want to exclude a tag from sorting, add `<! -- nolint -->` at the beginning of the line.
```

> Example

Reviewer: [IMO] This is redundant.

```suggestion
```

Reviewer: Or, writing like this (`, for example:`) looks good.

https://github.com/pre-commit/pre-commit-hooks#sort-simple-yaml
